### PR TITLE
[cherry-pick for release-1.9]fix jobflow CRD metadata.annotations: Too long error

### PR DESCRIPTION
**Pull request by guoqinwill**
cherry-pick：https://github.com/volcano-sh/volcano/pull/3347

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -114,6 +114,8 @@ manifests: controller-gen
 	$(CONTROLLER_GEN) "crd:maxDescLen=0,crdVersions=v1beta1" paths="./vendor/volcano.sh/apis/pkg/apis/batch/v1alpha1" output:crd:artifacts:config=config/crd/volcano/v1beta1
 	# jobflow crd base
 	$(CONTROLLER_GEN) $(CRD_OPTIONS) paths="./vendor/volcano.sh/apis/pkg/apis/flow/v1alpha1" output:crd:artifacts:config=config/crd/jobflow/bases
+  # generate volcano jobflow crd yaml without description to avoid yaml size limit when using `kubectl apply`
```
Comment (shinytang6) [inline Makefile:117]: pls format the comments

**Review by shinytang6 — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (guoqinwill) [reply Makefile]: done

**Review by william-wang — APPROVED** [review]

**Pull request merged**